package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@magnet.me/consultant-js",
+  "version": "1.0.0",
+  "description": "A library to register services to Consul and listen for property changes.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Alex Kolpa",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "atob": "^2.0.3",
+    "request": "^2.81.0",
+    "uuid": "^3.0.1"
+  },
+  "devDependencies": {
+    "babel-preset-env": "^1.5.1",
+    "express": "^4.15.3"
+  },
+  "directories": {
+    "example": "example"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Magnetme/consultant-js.git"
```

Comment:
~~easier to only list https. People without certificates installed cannot access the ssh version~~ nvm thats what you did